### PR TITLE
Adds support for app lists

### PR DESCRIPTION
**Pull request by berndverst**
@brandonblack 

Resolves feature work for issue #14

### CODE REVIEW

File: spec/fixtures/app_lists_load.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+         {
+            "os_type" : "Android",
+            "app_store_identifier" : "com.supercell.clashofclans"
+         },
+         {
+            "app_store_identifier" : "com.functionx.viggle",
+            "os_type" : "Android"
+         },
+         {
+            "app_store_identifier" : "io.fabric.samples.cannonball",
```
Comment (andypiper) [inline spec/fixtures/app_lists_load.json]: the best app in the best app list ever?

Comment (brandonblack) [inline spec/fixtures/app_lists_load.json]: :tada:

File: spec/fixtures/app_lists_load.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "name" : "Some test app list",
+      "id" : "abc2"
+   },
+   "request" : {
+      "params" : {
+         "account_id" : "2iqph",
+         "app_list_id" : "abc2"
+      }
+   }
+}
```
Comment (brandonblack) [inline spec/fixtures/app_lists_load.json]: super minor, but do you mind adding a newline here to the EOF?

File: spec/fixtures/app_lists_all.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      {
+         "name" : "Yet another app list",
+         "id" : "wdpr"
+      },
+      {
+         "name" : "The best app list yet",
+         "id" : "wdps"
+      }
+   ]
+}
```
Comment (brandonblack) [inline spec/fixtures/app_lists_all.json]: newline here as well

File: lib/twitter-ads/app_list.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    RESOURCE_COLLECTION = '/0/accounts/%{account_id}/app_lists' # @api private
+    RESOURCE            = '/0/accounts/%{account_id}/app_lists/%{id}' # @api private
+
+    def initialize(account)
+      @account = account
+      self
+    end
+
+    # Creates a new App List
```
Comment (brandonblack) [inline lib/twitter-ads/app_list.rb]: newline after this first line

File: lib/twitter-ads/app_list.rb [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    # @param name [String] The name for the app list to be created.
+    # @param *ids [String] String or String Array of app IDs.
```
Comment (brandonblack) [inline lib/twitter-ads/app_list.rb]: you can also make this `ids`

File: spec/fixtures/app_lists_load.json (hunk 1 above)
Comment (berndverst) [inline spec/fixtures/app_lists_load.json]: Indeed.

File: spec/fixtures/app_lists_load.json (hunk 2 above)
Comment (berndverst) [inline spec/fixtures/app_lists_load.json]: Done

File: spec/fixtures/app_lists_all.json (hunk 3 above)
Comment (berndverst) [inline spec/fixtures/app_lists_all.json]: Done

File: lib/twitter-ads/app_list.rb (hunk 4 above)
Comment (berndverst) [inline lib/twitter-ads/app_list.rb]: Done

**Comment by brandonblack** [comment]
Looks fantastic. Thanks @berndverst! :shipit:

**Pull request merged**